lib/builtins/deploy-delegates/lambda-deployer/helper.js

```diff
@@ -10,56 +10,86 @@ const retryUtils = require('@src/utils/retry-utility');
 const stringUtils = require('@src/utils/string-utils');
 
 module.exports = {
-    validateLambdaDeployState,
+    loadLambdaInformation,
     deployIAMRole,
     deployLambdaFunction
 };
 
-function validateLambdaDeployState(reporter, awsProfile, awsRegion, currentRegionDeployState, ignoreHash, callback) {
-    const lambdaData = currentRegionDeployState.lambda;
-    if (R.isNil(lambdaData) || R.isEmpty(lambdaData) || R.isNil(lambdaData.arn)) {
-        return callback(null, { updatedDeployState: currentRegionDeployState });
+function loadLambdaInformation(reporter, loadLambdaConfig, callback) {
+    const { awsProfile, awsRegion, alexaRegion, deployState, userConfig, ignoreHash } = loadLambdaConfig;
+    const lambdaClient = new LambdaClient({ awsProfile, awsRegion });
+    // check if user sets userConfig with "sourceLambda" field to reuse existing Lambda
+    const sourceLambdaArn = alexaRegion === 'default' ? R.view(R.lensPath(['sourceLambda', 'arn']), userConfig)
+        : R.view(R.lensPath(['regionalOverrides', alexaRegion, 'sourceLambda', 'arn']), userConfig);
```

Comment:
optional(can be fixed during rewrite of lambda deployer)
can just use 
https://ramdajs.com/docs/#path

similar to other places.